contracts/astroport-liquidity-helper/tests/astroport_integration_tests.rs

```diff
@@ -452,3 +516,26 @@ where
     .unwrap()
     .balance
 }
+
+pub fn f64_to_dec<T>(val: f64) -> T
+where
+    T: FromStr,
+    T::Err: std::error::Error,
+{
+    T::from_str(&val.to_string()).unwrap()
+}
+
+pub fn common_pcl_params() -> ConcentratedPoolParams {
+    ConcentratedPoolParams {
+        amp: f64_to_dec(40f64),
+        gamma: f64_to_dec(0.000145),
+        mid_fee: f64_to_dec(0.0026),
+        out_fee: f64_to_dec(0.0045),
+        fee_gamma: f64_to_dec(0.00023),
+        repeg_profit_threshold: f64_to_dec(0.000002),
+        min_price_scale_delta: f64_to_dec(0.000146),
+        price_scale: Decimal::one(),
+        ma_half_time: 600,
+        track_asset_balances: None,
+    }
+}
```

Comment:
are these close to what will be used for the wstETH/USDC pool?

Reply:
Tbh I have no idea what they will use for that pool. I just copied these from astroports tests.